giskard/project.py

```diff
@@ -437,15 +440,27 @@ def validate_columns_columntypes(df: pd.DataFrame, column_types) -> pd.DataFrame
             return df
 
     @staticmethod
-    def _verify_category_columns(df: pd.DataFrame, column_types):
+    def _validate_category_columns(df: pd.DataFrame, column_types):
         for name, types in column_types.items():
             if types == SupportedColumnType.CATEGORY.value and len(df[name].unique()) > 30:
                 warnings.warn(f"Categorical feature '{name}' contains {len(df[name].unique())} distinct values. If "
                               f"necessary use 'numeric' or 'text' in column_types instead")
 
     @staticmethod
-    def _verify_is_pandasdataframe(df):
+    def _validate_is_pandasdataframe(df):
         assert isinstance(df, pd.DataFrame), "Dataset provided is not a pandas dataframe"
 
+    @staticmethod
+    def _run_sample_prediction(small_df, prediction_function):
```

Comment:
why do we need this method?

Reply:
We actually don't need this. I have updated the code. This one was useful when I was calling the prediction twice and I had hard-coded the number of rows inside the function.  This code is refactored now.